airstack-modules/modules/airstack/social/social.ts

```diff
@@ -723,8 +726,228 @@ export namespace social {
       airSocialProfile
     )
   }
+
+/**
+ * @dev This function tracks profile Handle registration transaction
+ * @param block ethereum block
+ * @param handleId Id of handle
+ * @param to Owner of handle
+ * @param tokenId TokenId of Handle
+ * @param tokenAddress  TokenAddress of Handle
+ * @param name  Name of Handle
+ * @param nameSpace Name space of Handle
+ */
+export function trackSocialProfileHandleRegistrationTransaction(
+  block: ethereum.Block,
+  to: string,
+  tokenId: string,
+  tokenAddress: string,
+  name: string,
+  nameSpace: string,
+): void {
+
+  const chainId = getChainId()
+  const id = getProfileHandleId(chainId,tokenAddress, tokenId);
+  let entity = AirSocialProfileHandle.load(id);
+
+  if(entity == null) {
+    entity = new AirSocialProfileHandle(id);
+  }
+
+  const airBlock = getOrCreateAirBlock(
+    chainId,
+    block.number,
+    block.hash.toHexString(),
+    block.timestamp
+  )
+  airBlock.save();
+  const airAccount = getOrCreateAirAccount(chainId, to, airBlock)
+  airAccount.save();
+
+  const token = getOrCreateAirToken(chainId, tokenAddress);
+  token.save();
+
+  entity.createdAt = airBlock.id;
+  entity.owner = airAccount.id;
+  entity.name = name;
+  entity.namespace = nameSpace;
+  entity.tokenAddress = token.id;
+  entity.tokenId = tokenId;
+  saveProfileHandle(entity, airBlock)
+}
+
+/**
+ * @dev This function tracks profile Handle transfer transaction
+ * @param block ethereum block
+ * @param handleId Id of handle
+ * @param from current Owner of handle
+ * @param to New owner of handle
+ * @param tokenId TokenId of Handle
+ * @param tokenAddress  TokenAddress of Handle
+ 
+ */
+export function trackSocialProfileHandleTransferTransaction(
+  block: ethereum.Block,
+  from: string,
+  to: string,
+  tokenId: string,
+  tokenAddress: string,
+  
+): void {
+
+  const chainId = getChainId()
+  const id = getProfileHandleId(chainId, tokenAddress, tokenId);
+  let entity = AirSocialProfileHandle.load(id);
+
+  if(entity == null) {
+    throw new Error("air social profile handle not found")
+  }
+
+  const airBlock = getOrCreateAirBlock(
+    chainId,
+    block.number,
+    block.hash.toHexString(),
+    block.timestamp
+  )
+  airBlock.save();
+  const airAccount = getOrCreateAirAccount(chainId, to, airBlock)
+  airAccount.save();
+
+  entity.owner = airAccount.id;  
+
+  // if handle is burned, unlink it with profile if already linked
+  if(to == ZERO_ADDRESS && entity.profile != null) {
```

Comment:
should we consider only zero address or also maybe address like 0x...01

Reply:
I think protocol consider 0x00..00 as burn address. 
Check the comment here. 
https://github.com/lens-protocol/core/blob/1c2ea835a468588c0d58389a044105e26b224b39/contracts/namespaces/constants/Events.sol#L32